browser/src/Services/BufferUpdates.ts

```diff
@@ -52,5 +52,15 @@ export class BufferUpdates {
                 this._pluginManager.notifyBufferUpdate(args, bufferLines)
             }
         })
+        
+        this._neovimInstance.on("buffer-update-incremental", (args: Oni.EventContext, bufferLine: string, lineNumber: number) => {
+            // const lastLine = args.line
+            // this._lastArgs = args
+            // this._lastBufferLines = bufferLine
+            // this._lastBufferVersion = args.version
+
+            // If we can send incremental updates, and the line hasn't changed, just send the incremental change
```

Comment:
I'm not sure what of this I should keep in... it only receives one line, whereas the whole buffer update takes them all